### ISSUE TITLE
I hope this is the right place

### ISSUE THREAD

**Issue by kk80586** [deeponion/deeponion-legacy#48]
I am having issues... 

/home/kevin/deeponion/src/db.h:161: undefined reference to `Dbt::Dbt(void*, unsigned int)'
/home/kevin/deeponion/src/db.h:167: undefined reference to `Dbt::Dbt(void*, unsigned int)'
/home/kevin/deeponion/src/db.h:167: undefined reference to `Dbt::~Dbt()'
/home/kevin/deeponion/src/db.h:161: undefined reference to `Dbt::~Dbt()'
/home/kevin/deeponion/src/db.h:167: undefined reference to `Dbt::~Dbt()'
/home/kevin/deeponion/src/db.h:161: undefined reference to `Dbt::~Dbt()'
collect2: error: ld returned 1 exit status
Makefile:2131: recipe for target 'DeepOniond' failed
make[1]: *** [DeepOniond] Error 1
make[1]: Leaving directory '/home/kevin/deeponion/src'
Makefile:525: recipe for target 'all-recursive' failed
make: *** [all-recursive] Error 1
*****************************************************************************************************
I don't know if these were the only issues as I get a lot of warnings:
(Example-    leveldb/util/logging.cc:58:40: warning: comparison between signed and unsigned integer expressions [-Wsign-compare])

Debian 8.8 (Jessie)
Intel CORE i5
Wallet 1.4.3 installed (fairly smoothly)

I can supply more info and output but don't want to spam you with unneeded info. Please let me know any info you need. Thanks, Kevin

**Comment by dragononcrypto** [comment]
Could you provide the results of configure? I'm not one of the developers here but I'm building in a version of Debian stretch at the moment. Also how did you install Berekeley 4.8? This appears to be the problem.

**Comment by brakmic** [comment]
Debian is not tested by us and it's not a secret that Debian distros often have outdated or otherwise "different" libraries.

Maybe you could help yourself by compiling the Berkeley DB. For example, [this tutorial](https://github.com/bitcoin/bitcoin/blob/0.9/doc/build-unix.md#berkeley-db) from Bitcoin Core is easy to follow.

And. please, DO NOT use stupid titles like "I hope this, that, maybe blah blah".

GitHub is **serious** place for **serious** issues. 

If you have *configuration problems*, please, **FIRST** do a **Google Search** before coming to us to drain our resources.

Thanks!

**Comment by kk80586** [comment]
Hope this works. I think I used pastebin once years ago-- output from sudo ./configure --with-gui=qt5 = https://pastebin.com/kuQBxtYh
Let me know if that doesn't work.
How did I install Berkeley4.8? VERY good question :P  
I did purge the 5.3 per instructions here:
https://deeponion.org/community/threads/tutorial-installing-deeponion-v1-5-1-on-ubuntu-debian.28202/
Don't know if that helped or not.
I installed in ~/  as well as  /usr/local  because I read somewhere that it was better to install in userspace.
some of the things I did (possibly not in order)
cd db-4.8.30.NC
../dist/configure  --prefix=/usr/local      <-- I believe I tried this first

../dist/configure --disable-replication  --enable-cxx --prefix=/usr/local     <--- I think this one did it?

I believe I needed to set the LIBRARY Path:  (system may have done that since I see it says /usr/local  and I edited   /etc/ld.so.conf.d/pico x86_64-linux-gnu.conf  to be /home/kevin/BerkeleyDB.4.8/lib
$ env | grep '^LD_LIBRARY_PATH'   --should yield-->  LD_LIBRARY_PATH=:/usr/local/BerkeleyDB.4.8/lib
I've tried a lot of things over the past several days so if I think of anything else I'll you know. I wish I knew why my bash history is not retaining after reboot. Will have to look into that. I also have pages bookmarked and will let you know some pages I visited and might have used info it contained. I will get those together and email those to you. (I like your email handle :)  )
